### ISSUE TITLE
Station ID 611 (Times sq) does not display 

### ISSUE THREAD

**Issue by dustbits** [Elaniobro/MMM-nyc-transit#78]
Hello!

I am having an issue with station ID 671, it does not display regardless of the values I set in the config. My current entry for is looks like this : 

          stationId: 671,
          walkingTime: 10,
          dir: {
              upTown: true,
              downTown: false

I changed the stationID value to that of other stations and they display without a hitch. I also stationID 471 is doing the same thing. 

StationID 671
![1](https://user-images.githubusercontent.com/43354758/153916697-6381a341-5772-4ede-9b9d-7fc28990c23b.JPG)

Different station
![image](https://user-images.githubusercontent.com/43354758/153916871-250aebc6-6cfd-4d7d-ab0c-3ba7ce4301ff.png)

**Comment by Elaniobro** [comment]
@dustbits `671` does not appear to be a station ID in the [Station List](https://github.com/Elaniobro/MMM-nyc-transit/blob/master/STATION_LIST.md) 

did you mean `611` for:

```js
{
    "id": : 611,
    "name": "Times Sq - 42 St / Port Authority Bus Terminal"
  },
  ```

**Comment by dustbits** [comment]
Sorry that was my mistake, I changed it to 611 however and still having the same problem

**Comment by dustbits** [comment]
Any chance you are able to test 611 in your build?

**Comment by Elaniobro** [comment]
@dustbits; I saw you closed this issue. All good?

**Comment by dustbits** [comment]
Hey,

Thanks for following up, actually I closed it because the other two issues open are the same as mine and I'm following those threads. Only my stations (Times square & hudson yards) are having these issues unfortunately, the other ones I tested work fine. I'll follow up closely cause I just built my mirror last month and this is definitely a module I'd love to have.

Thanks for building this!

Sent with [ProtonMail](https://protonmail.com/) secure email.

------- Original Message -------
